app/models/concerns/awardee.rb

```diff
@@ -0,0 +1,27 @@
+module Awardee
+  def acquired_medals
+    medals_for all_medals.first
+  end
+
+  def unacquired_medals
+    medals_for all_medals.second
+  end
+
+  private
+
+  def medals_for(indicators)
+    indicators.map { |i| i.content.medal }.compact
+  end
+
+  def all_medals
+    current_indicators.partition(&:once_completed?)
+  end
+
+  def current_indicators
+    items_with_medals.map { |i| i.progress_for(self, Organization.current) }
+  end
+
+  def items_with_medals
+    Usage.where(organization: Organization.current).map(&:item).select(&:medal_id)
+  end
```

Comment:
The only (I believe) thing missing is how to improve this query, or if it at least we can avoid querying twice for acquired and unacquired medals. This might have to be solved through Laboratory - I'll add whatever I come up with on  https://github.com/mumuki/mumuki-laboratory/pull/1469